### PR TITLE
Add azure release 15.1.3

### PR DESCRIPTION
**Pull request by whites11**
Azure release 15.1.3 with backport of changes in 16.0.2

### CODE REVIEW

File: azure/requests.yaml [hunk 1]
```diff
@@ -11,96 +11,3 @@ releases:
           version: ">= 1.8.0"
         - name: cert-operator
           version: "< 1.2.0"
-    - name: "> 15.0.1"
```
Comment (whites11) [inline azure/requests.yaml:14]: Cleaned up old requests

File: azure/v16.0.2/README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ on the workloads should be as good as it can be.
 - Updates golang.org/x/text to v0.3.6 to fix CVE-2020-28852 (#102601, @jonesbr17) [SIG API Machinery, CLI, Cloud Provider, Cluster Lifecycle, Instrumentation, Node and Storage]
 
 
-### azure-operator [5.10.0](https://github.com/giantswarm/cert-operator/releases/tag/v1.2.0)
```
Comment (whites11) [inline azure/v16.0.2/README.md:34]: wrong link in previous release

**Review by fiunchinho — APPROVED** [review]

**Review by T-Kukawka — APPROVED** [review]

**Pull request merged**